sdk/storage/azblob/CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release History
 
-## 0.5.0 (Unreleased)
+## 0.5.0 (Beta)
 
 ### Breaking Changes
 
-* Complete architectural change for better user experience. Please see the example file for more information.
+* Complete architectural change for better user experience. Please view the [README](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azblob#readme)
+* Added [UserDelegationCredential](https://learn.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas) which resolves [#18976](https://github.com/Azure/azure-sdk-for-go/issues/18976), [#16916](https://github.com/Azure/azure-sdk-for-go/issues/16916), [#18977](https://github.com/Azure/azure-sdk-for-go/issues/18977) 
```

Comment:
This bullet, and the next, aren't breaking changes so they should go under a `### Features Added` header.